cmd/proxy/config/swagger/openapi.json

```diff
@@ -2985,6 +3020,51 @@
           }
         }
       },
+      "Guardian": {
+        "type": "object",
+        "properties": {
+          "address": {
+            "type": "string"
+          },
+          "activationEpoch": {
+            "type": "integer"
+          },
+          "serviceUID": {
+            "type": "string"
+          }
+        }
+      },
+      "GuardianResponse": {
+        "type": "object",
+        "properties": {
+          "data": {
+            "type": "object",
+            "properties": {
+              "activeGuardian": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/components/schemas/Guardian"
+                }
+              },
+              "pendingGuardian": {
+                "type": "array",
```

Comment:
```suggestion
                "type": "object",
```

Reply:
it won't work (it won't display the fields inside the object)